beitragsordnung.md

```diff
@@ -2,10 +2,12 @@
 
 ## § 1 Höhe des Mitgliedsbeitrags
 
-1. Der jährliche Mitgliedsbeitrag beträgt für ordentliche Mitglieder,
-    1. welche natürliche Personen sind: 12 €,
-    2. welche keine natürlichen Personen sind: 1337€.
-1. Der Vorstand kann auf Antrag eine Befreiung von der Zahlungspflicht, oder eine Ermäßigung des Beitrages erteilen, ggf. auch rückwirkend. Der Vorstand entscheidet nach billigem Ermessen.
+1. Der jährliche Mitgliedsbeitrag beträgt für ordentliche Mitglieder, welche laut Satzung (§3 Abs. 1) natürliche Personen sind: 12€
+2. Der Vorstand kann auf Antrag eine Befreiung von der Zahlungspflicht, oder eine Ermäßigung des Beitrages erteilen, ggf. auch rückwirkend. Der Vorstand entscheidet nach billigem Ermessen.
+3. Der jährliche Mitgliedsbeitrag beträgt für Fördermitglieder,
+    1. welche natürliche Personen sind: mindestens 120€,
```

Comment:
```suggestion
    1. welche natürliche Personen sind: mindestens 60€,
```
10€ im Monat ist eigentlich nicht zu viel, wir sollten das als den "normalen" Beitrag für Fördermitgliedschaftsanträge setzen, aber besonders da wir das ja vor haben für das ganze Jahr abzubuchen, und 120€ auf ein mal auch für 27 jährige (bzw. >25 da die nicht mehr ordentliche Mitglieder werden können) noch ein beachtlicher Betrag sein kann, wäre ich dafür den minimalbetrag als Kompromiss auf 60€ zu setzen.

Reply:
Ich würde vorschlagen, dass die Beiträge Quartalsweise eingezogen werden. Dann reden wir über 30€ im Quartal.

Zumal der Vorstand noch die Option hat, die Beiträge auf Anfrage des Mitgliedes zu senken.

Da Fördermitglieder unsere primäre (bzw. einzig signifikante) Geldquelle sind, würde ich den Standart-Betrag nicht von vorne herein schon so stark senken.

Reply:
Zehn Euro im Monat sind im Vergleich zu Beiträgen in anderen Vereinen (Normalbetrag üblicherweise 20-30 EUR) wahnwitzig Gering für **Förder**mitglieder, deren Zweck es ist, den Verein finanziell und nicht gestalterisch zu tragen.